test/unit/ssz_ex_test.exs

```diff
@@ -12,6 +12,493 @@ defmodule Unit.SSZExTest do
     assert {:error, ^error_message} = SszEx.decode(serialized, schema)
   end
 
+  test "packing a list of uints" do
+    list_1 = [1, 2, 3, 4, 5]
+
+    expected_1 =
+      <<1, 2, 3, 4, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0>>
+
+    actual_1 = SszEx.pack(list_1, {:list, {:int, 8}, 5})
+    assert expected_1 == actual_1
+
+    list_2 = [
+      18_446_744_073_709_551_595,
+      18_446_744_073_709_551_596,
+      18_446_744_073_709_551_597,
+      18_446_744_073_709_551_598,
+      18_446_744_073_709_551_599,
+      18_446_744_073_709_551_600,
+      18_446_744_073_709_551_601,
+      18_446_744_073_709_551_603,
+      18_446_744_073_709_551_604,
+      18_446_744_073_709_551_605,
+      18_446_744_073_709_551_606,
+      18_446_744_073_709_551_607,
+      18_446_744_073_709_551_608,
+      18_446_744_073_709_551_609,
+      18_446_744_073_709_551_610,
+      18_446_744_073_709_551_611,
+      18_446_744_073_709_551_612,
+      18_446_744_073_709_551_613,
+      18_446_744_073_709_551_614,
+      18_446_744_073_709_551_615
+    ]
+
+    expected_2 =
+      <<235, 255, 255, 255, 255, 255, 255, 255, 236, 255, 255, 255, 255, 255, 255, 255, 237, 255,
+        255, 255, 255, 255, 255, 255, 238, 255, 255, 255, 255, 255, 255, 255, 239, 255, 255, 255,
+        255, 255, 255, 255, 240, 255, 255, 255, 255, 255, 255, 255, 241, 255, 255, 255, 255, 255,
+        255, 255, 243, 255, 255, 255, 255, 255, 255, 255, 244, 255, 255, 255, 255, 255, 255, 255,
+        245, 255, 255, 255, 255, 255, 255, 255, 246, 255, 255, 255, 255, 255, 255, 255, 247, 255,
+        255, 255, 255, 255, 255, 255, 248, 255, 255, 255, 255, 255, 255, 255, 249, 255, 255, 255,
+        255, 255, 255, 255, 250, 255, 255, 255, 255, 255, 255, 255, 251, 255, 255, 255, 255, 255,
+        255, 255, 252, 255, 255, 255, 255, 255, 255, 255, 253, 255, 255, 255, 255, 255, 255, 255,
+        254, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255>>
+
+    actual_2 = SszEx.pack(list_2, {:list, {:int, 64}, 15})
+    assert expected_2 == actual_2
+  end
+
+  test "packing a list of booleans" do
+    list = [true, false, true, false, true]
+
+    expected =
+      <<1, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0>>
+
+    actual = SszEx.pack(list, {:list, :bool, 5})
+    assert expected == actual
+  end
+
+  test "packing a list of bytes" do
+    list_1 = [<<1>>, <<2>>, <<3>>, <<4>>, <<5>>]
+
+    expected_1 =
+      <<1, 2, 3, 4, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0>>
+
+    actual_1 = SszEx.pack(list_1, {:list, {:bytes, 8}, 5})
```

Comment:
The bytes type isn't actually a basic type, as it'd be defined as `{:vector, {:int, 8}, N}` (see #593). Let's just change this for something else.

Reply:
Should I just add a todo on this since it depends on other issues?

Reply:
I think we can safely remove this test.